services/account_type/cloudtrail_owner.yaml

```diff
@@ -0,0 +1,114 @@
+AWSTemplateFormatVersion: "2010-09-09"
+
+
+Description: "CloudZero :: CloudTrail owner template"
```

Comment:
Please follow the convention for `Description` in the other templates for consistency, something like `CloudZero Connected Account Template`